mix.exs

```diff
@@ -7,7 +7,7 @@ defmodule Commanded.EventStore.Adapters.EventStore.Mixfile do
     [
       app: :commanded_eventstore_adapter,
       version: @version,
-      elixir: "~> 1.9",
```

Comment:
The minimum version with [security support is 1.12](https://endoflife.date/elixir), so it doesn't make too much sense to support something that is no longer supported.